### ISSUE TITLE
Add deb package

### ISSUE THREAD

**Issue by CsatiZoltan** [bk138/gromit-mpx#146]
I successfully created a deb package with the following two steps:
1. Append the CMakeLists.txt file with
   ```
   SET(CPACK_GENERATOR "DEB")
   SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "bk138")
   INCLUDE(CPack)
   ```
2. After building the package (`make`), the .deb file can be created by
   ```
   cpack -G DEB
   ```

This packages the man pages, the desktop entry, etc.

**Comment by bk138** [comment]
If you open a pull request and add yourself as maintainer, we can add those bits.

**Comment by CsatiZoltan** [comment]
@bk138 
> add yourself as maintainer

Do you mean that I change `"bk138"` to my name in `SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "bk138")`?

In the meantime, I realized that I need to specify the version as well, otherwise CPack give the default `0.1.1`, which is lower than the version available in the [Ubuntu package index](https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=gromit-mpx&searchon=names), causing the Software Updater to want to upgrade it with the packaged versions. I will also add the dependencies. Anyway, I open a PR and we can discuss these issues there.

**Comment by bk138** [comment]
> Do you mean that I change `"bk138"` to my name in `SET(CPACK_DEBIAN_PACKAGE_MAINTAINER "bk138")`?

Yep, given the fact that I won't have time for Debian package maintenance in the foreseeable future *and* that of course you'd be willing to answer deb-related user questions that might arise. This is an invitation, if you will ;-)

**Comment by CsatiZoltan** [comment]
I have no experience with Debian packaging, but I take up the challenge. First, I try to solve the issues I have in mind to see if it works. If it does, I will submit a PR.

**Comment by bk138** [comment]
Tentatively closing this, please re-open or post here when there are new developments. In the meantime, y'all can use the flatpak on Linux.
